.github/workflows/ci.yml

```diff
@@ -10,7 +10,7 @@ on:
       - prerelease/*
   pull_request:
     branches:
-      - master
```

Comment:
Did you mean to include this change? 

Might be a little heavy on the processing, but maybe we can try it out and see how it goes (I'm not clear on our pipeline utilization actually).

Reply:
i actually reverted this again in a later PR. i don't think '*' works